packages/repository/test/acceptance/hasMany relation/hasMany-relation.acceptance.ts

```diff
@@ -21,27 +21,37 @@ describe('hasMany relationship', () => {
       name: 'John Smith',
       slug: 'johnsmith@yahoo.com',
     });
-    c1.orders.create({
+    const order = await c1.orders.create({
+      // customerId should be enforced
       desc: 'order1 description',
       date: new Date().toISOString(),
+    });
+
+    let foundOrder = await orderRepo.findById(order.id);
+    expect(foundOrder).to.have.properties({
+      desc: 'order1 description',
       customerId: c1.getId(),
     });
-    let found = await customerRepo.findById(c1.id);
-    expect(found).to.have.properties({
+
+    let foundCustomer = await customerRepo.findById(c1.id);
+    expect(foundCustomer).to.have.properties({
       name: c1.name,
       email: c1.email,
       orders: [
```

Comment:
I don't think `orders` will be included by default. It only happens when the `filter` has `includes: ['orders']`.

We also need to be careful about the name of the relational property. For example, `orders` is already used for the corresponding order repository with the FK. Using `orders` to include related orders can be confusing here.

Reply:
Good catch! I've renamed the relational property and added in the includes filter.